tests/certification/bindings/cron/cron_test.go

```diff
@@ -0,0 +1,306 @@
+/*
+Copyright 2021 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cron_test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/benbjohnson/clock"
+	"github.com/stretchr/testify/require"
+
+	"github.com/dapr/components-contrib/bindings"
+	"github.com/dapr/components-contrib/bindings/cron"
+	"github.com/dapr/dapr/pkg/runtime"
+	"github.com/dapr/go-sdk/service/common"
+	"github.com/dapr/kit/logger"
+
+	// Certification testing runnables
+	"github.com/dapr/components-contrib/tests/certification/embedded"
+	"github.com/dapr/components-contrib/tests/certification/flow"
+	"github.com/dapr/components-contrib/tests/certification/flow/app"
+	"github.com/dapr/components-contrib/tests/certification/flow/sidecar"
+
+	bindings_loader "github.com/dapr/dapr/pkg/components/bindings"
+	dapr_testing "github.com/dapr/dapr/pkg/testing"
+)
+
+type cronTest struct {
+	cronName                 string        // name of the cron binding
+	schedule                 string        // cron schedule
+	expectedTriggerCount     int           // expected number of triggers within the deadline
+	timeoutToObserveTriggers time.Duration // time to add to the mock clock to observe triggers
+	startTime                time.Time     // starting time for the mock clock
+	clk                      *clock.Mock   // mock clock
+}
+
+// Test cron triggers with different schedules
+func TestCronBindingTrigger(t *testing.T) {
+
+	appName := "cronapp"
+	sidecarName := "cron-sidecar"
+
+	testMatrix := []cronTest{
+		{
+			cronName:                 "cron1s",
+			schedule:                 "@every 1s", // Test macro cron format
+			expectedTriggerCount:     10,
+			timeoutToObserveTriggers: time.Second * 10,
+			startTime:                time.Now(),
```

Comment:
nit: since it's a mock clock, maybe you can use a constant here like `time.UnixDate`.

Reply:
using a constant time for all test cases